src/recommendationservice/recommendation_server.py

```diff
@@ -6,19 +6,23 @@
 
 # Python
 import os
+import sys
 import random
 from concurrent import futures
 
+# New Relic
+import newrelic.agent
+
 # Pip
 import grpc
-from opentelemetry import trace, metrics
-from opentelemetry._logs import set_logger_provider
-from opentelemetry.exporter.otlp.proto.grpc._log_exporter import (
-    OTLPLogExporter,
-)
-from opentelemetry.sdk._logs import LoggerProvider, LoggingHandler
-from opentelemetry.sdk._logs.export import BatchLogRecordProcessor
-from opentelemetry.sdk.resources import Resource
+# from opentelemetry import trace, metrics
```

Comment:
Instead of commenting out code it could be cool to use a feature flag to gate whether to use OpenTelemetry or the NR agent. The demo environment has a great feature flag service you could leverage.